solutions/usaco-1061.mdx

```diff
@@ -106,4 +106,131 @@ for x in eat:
 
 </PySection>
 
+<JavaSection>
+```java
+
+import java.io.*;
+import java.util.*;
+
+public class StuckInARut {
+	public static void main(String[] args) throws IOException {
+		BufferedReader r = new BufferedReader(
+		new InputStreamReader(System.in));
+		PrintWriter pw = new PrintWriter(System.out);
+		StringTokenizer st = new StringTokenizer(r.readLine());
+
+		int n = Integer.parseInt(st.nextToken());
+		List<int[]> northCows = new ArrayList<>();
+		List<int[]> eastCows = new ArrayList<>();
+
+		for (int i = 0; i < n; i++) {
+			st = new StringTokenizer(r.readLine());
+			String dir = st.nextToken();
+			int x = Integer.parseInt(st.nextToken());
+			int y = Integer.parseInt(st.nextToken());
+
+			/*
+			* Also append i since we have to return them
+			* in the same order they were given to us.
+			*/
+			if (dir.equals("N")) {
+				northCows.add(new int[]{x, y, i});
+			} else {
+				eastCows.add(new int[]{x, y, i});
+			}
+		}
+		/*
+		* https:docs.oracle.com/javase/8/docs/api/java/util/Comparator.html
```

Comment:
```suggestion
		* https://docs.oracle.com/javase/8/docs/api/java/util/Comparator.html
```
link supposed to be like this. (optional)
(web browsers automatically add the two slashes)